_world_sessions/2024/day-2/closing-keynote-aaron.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: world/2024/session
+title: Closing Keynote
+speaker: aaron-patterson.md # name of md speaker file
+<<<<<<< HEAD
+=======
+<<<<<<<< HEAD:_world_sessions/2024/day-2/aaron-patterson.md
+>>>>>>> 181911d (Refactored the page)
+<<<<<<<< HEAD:_world_sessions/2024/day-2/closing-keynote.md
+time: 16:30 - 17:30
+location: Track 1
+========
+time: 16:00 - 19:00
+location: 16:30 - 17:30
+>>>>>>>> 2b73e36 (Updated the Agenda):_world_sessions/2024/day-2/aaron-patterson.md
+<<<<<<< HEAD
+=======
+========
+time: 16:30 - 17:30
+location: Track 1
+>>>>>>>> 181911d (Refactored the page):_world_sessions/2024/day-2/closing-keynote.md
+>>>>>>> 181911d (Refactored the page)
+running_order: 12
+---
+
+More information coming soon.
```

Comment:
@akhilgkrishnan Not exactly sure which is the correct data or not with this - It might've been a GitHub issue the way this file got saved (this is how the file actually is in the codebase)

Reply:
Is this how it should be? 

```md
---
layout: world/2024/session
title: Closing Keynote
speaker: aaron-patterson.md # name of md speaker file
time: 16:30 - 17:30
location: Track 1
running_order: 12
---

More information coming soon.
```

Reply:
Ohh sorry, that one is merge conflict.